code/go/pkg/validator/validator_test.go

```diff
@@ -28,9 +29,6 @@ func TestValidateFile(t *testing.T) {
 		"deploy_terraform":    {},
 		"time_series":         {},
 		"missing_data_stream": {},
-		"custom_logs":         {},
-		"httpjson_input":      {},
-		"sql_input":           {},
```

Comment:
Could we keep these packages here if we are not enabling errors as warnings here?

Reply:
If I keep those packages here, they are going to be checked twice. 
In the new test suite where warnings are considered as errors too, if there is any error that is raised (and not expected), then the test should fail too as they do no expect any error (`{}`).

For instance, I added a new field `foo` in the manifest in one of them:
```
=== RUN   TestValidateWarnings/custom_logs
    validator_test.go:334: 
        	Error Trace:	validator_test.go:334
        	Error:      	"found 2 validation errors:
        	            	   1. Warning: package with non-stable semantic version and active beta features (enabled in [../../../../test/packages/custom_logs]) can't be released as stable version.
        	            	   2. file "../../../../test/packages/custom_logs/manifest.yml" is invalid: field (root): Additional property foo is not allowed
        	            	" should have 1 item(s), but has 2
        	Test:       	TestValidateWarnings/custom_logs
```
I could keep them in this test suite, in case the new test suite gets removed after structured errors are implemented. WDYT ? @jsoriano

Reply:
Ok, you are right, it is probably fine as it is. If we add structured errors we probably need to revisit these tests in any case.